sandbox/apps/sandbox/fields.js

```diff
@@ -16,9 +16,6 @@ module.exports = {
   },
   name: {
     validate: ['required', 'notUrl', { type: 'maxlength', arguments: 200 }],
-    // need to remove this for the heading to go
-    labelClassName: ['govuk-label--l'],
-    isPageHeading: 'true'
```

Comment:
I think I added this so it could be a heading, is there a reson for removing this?